src/app/_models/grade_book.ts

```diff
@@ -5,7 +5,7 @@ export type GradeBook = {
     event_name: string
     question_details: {
         title: string
-        question_grade: number
+        question_grade: string
```

Comment:
roll this back after the change in the back end PR. The front-end should be responsible for displaying with correct format not back-end.